spec/controllers/tags_controller_spec.rb

```diff
@@ -173,6 +173,66 @@
     end
   end
 
+  describe "show_hidden" do
+    render_views
```

Comment:
We generally avoid `render_views`. If we need to inspect the final HTML, we prefer Cucumber tests, which run a headless browser, even more so when the output involves JavaScript.

There's a Cucumber test that checks if the "Show warnings" and "Show additional tags" buttons work correctly:
https://github.com/otwcode/otwarchive/blob/3b3262673b355ff1980e57b21d300de4b576235b/features/other_a/preferences_edit.feature#L117-L130

You can extend the test to follow the links and check if you end up on the right works paths (`Then I should be on the works tagged "???"`).

The only RSpec test we should keep is the one for "redirects to referer with an error for HTML format".

Reply:
Okay, can do! I didn't realize there was already a test for this! Thanks.

Reply:
Removed render_views in 0b9be2643d228c57cf4e0960c899ea173adfbc3f. Extended preferences_edit.feature cucumber tests in 74fbc36c03483f2ad945b34e137e9b325b33c066 – but in local testing, I'm getting a failure because `Scary tag` is (sometimes?) not common enough to support a works page.

Reply:
Left a longer version of the above comment below - resolving this conversation for now.